<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1264</width>
    <height>1101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>661</width>
      <height>1061</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>601</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_OpenGL"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1264</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="New_menu"/>
    <addaction name="Open_menu"/>
    <addaction name="Exit_menu"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="RT60_menu"/>
    <addaction name="SPL_menu"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="New_menu">
   <property name="text">
    <string>NEW</string>
   </property>
  </action>
  <action name="Open_menu">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="Exit_menu">
   <property name="text">
    <string>EXIT</string>
   </property>
  </action>
  <action name="SPL_menu">
   <property name="text">
    <string>SPL CALCULATOR</string>
   </property>
  </action>
  <action name="RT60_menu">
   <property name="text">
    <string>RT 60 CALCULATOR</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
